user/README.md

```diff
@@ -1,80 +1,126 @@
-#Members
+#Users
 
-- Member object
+- User object
 
 ```
 {
-	"id": string,
-	"first_name": string,
-	"last_name": string,
-	"yoe": number,
-	"company": string,,
-	"designation": string,
-	"img": string,
-	"github_id": string,
-	"linkedin_id": string,
-	"twitter_id": string,
-  "instagram_id": string
+	'id': string,
+	'first_name': string,
+	'last_name': string,
+	'yoe': number,
+	'company': string,,
+	'designation': string,
+	'img': string,
+	'github_id': string,
+	'linkedin_id': string,
+	'twitter_id': string,
+  'instagram_id': string,
+  'site': string,
+  'github_display_name': string,
+  'isMember': boolean
 }
 ```
 
-## **GET /member**
+## **GET /user**
 
-Returns all members in the system.
+Returns all users in the system.
 
-- **URL Params**  
+- **Params**  
   None
+- **Query**  
+  size=[integer], page=[integer]
 - **Body**  
   None
 - **Headers**  
   Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
 - **Success Response:**
 - **Code:** 200  
   **Content:**
 
 ```
 {
-  message: "Members returned successfully!"
-  members: [
-           {<member_object>},
-           {<member_object>},
-           {<member_object>}
+  message: 'Users returned successfully!'
```

Comment:
Let's call it 'msg' to keep it consistent with the other API

Reply:
I case of error i.e. 4XX
if returns `{ error: 'Not Found', message: 'No users available' }`
which is being handled by boom (an npm package) so I thought I'll keep it same.

Reply:
Great point. Then let's change the other one to "message" in the future